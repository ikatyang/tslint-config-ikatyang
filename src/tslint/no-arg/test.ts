// good

function good1() {
  good1();
}

// bad

function bad1() {
  arguments.callee();
}
