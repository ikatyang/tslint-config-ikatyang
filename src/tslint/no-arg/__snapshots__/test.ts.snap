// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

function good1() {
  good1();
}

// bad

function bad1() {
  arguments.callee();
  ~~~~~~~~~~~~~~~~    [Access to arguments.callee is forbidden]
}

`;
